consent/models.py

```diff
@@ -239,3 +245,151 @@ class Meta(Consent.Meta):
         help_text=_("Course key for which data sharing consent is granted.")
     )
     history = HistoricalRecords()
+
+
+@python_2_unicode_compatible
+class DataSharingConsentPage(TimeStampedModel):
+    """
+    Stores texts to be used in data sharing consent page.
+    """
+
+    class Meta(object):
+        app_label = 'consent'
+
+    TOP_PARAGRAPH_HELP_TEXT = mark_safe_lazy(_(
+        'Fill in a text for first paragraph of page. The following variables may be available:<br />'
+        '<ul>'
+        '<li>enterprise_customer_name: A name of enterprise customer.</li>'
+        '<li>platform_name: Name of platform.</li>'
+        '</ul>'
+    ))
+    POLICY_PARAGRAPH_HELP_TEXT = mark_safe_lazy(_(
+        'Fill in a text for policy paragraph at the bottom of page. The following variables may be available:<br />'
+        '<ul>'
+        '<li>enterprise_customer_name: A name of enterprise customer.</li>'
+        '<li>platform_name: Name of platform.</li>'
+        '</ul>'
+    ))
+    SIDEBAR_PARAGRAPH_HELP_TEXT = mark_safe_lazy(_(
+        'Fill in a text for left sidebar paragraph. The following variables may be available:<br />'
+        '<ul>'
+        '<li>enterprise_customer_name: A name of enterprise customer.</li>'
+        '<li>platform_name: Name of platform.</li>'
+        '</ul>'
+    ))
+    CONFIRMATION_MODAL_HELP_TEXT = mark_safe_lazy(_(
+        'Fill in a text for dialog which appears when user decline to provide consent. '
+        'The following variables may be available:<br />'
+        '<ul>'
+        '<li>enterprise_customer_name: A name of enterprise customer.</li>'
+        '<li>item: A string which is "course" or "program" depending on the type of consent.</li>'
+        '</ul>'
+    ))
+
+    page_title = models.CharField(
+        max_length=255,
+        default=_('Data sharing consent required'),
+        help_text=_('Title of page')
+    )
+    left_sidebar_text = models.TextField(
+        null=True,
+        blank=True,
+        default=_(
+            "<p>Welcome to {platform_name}</p>"
+            "<p>{enterprise_customer_name} has partnered with {platform_name} to offer you high-quality learning "
+            "opportunities from the world's best universities.</p>"
+        ),
+        help_text=SIDEBAR_PARAGRAPH_HELP_TEXT
+    )
+    top_paragraph = models.TextField(
+        null=True,
+        blank=True,
+        default=_('<h2>Consent to share your data</h2>'
+                  '<p>To access this {item}, you must first consent to share your learning achievements with '
+                  '<b>{enterprise_customer_name}</b></p>'
+                  '<p>{enterprise_customer_name} would like to know about:</p>'
+                  '<ul><li>your enrollment in this course</li><li>your learning progress</li>'
+                  '<li>course completion</li></ul>'),
```

Comment:
Same as above, try to match default style:
```
<h2 class="consent-title">Consent to share your data</h2>
<p>To access this {item}, you must first consent to share your learning achievements with 
<b>{enterprise_customer_name}</b></p>
<p>{enterprise_customer_name} would like to know about:</p>
<ul><li>your enrollment in this course</li><li>your learning progress</li>
<li>course completion</li></ul><p>
```
make sure to add trailing paragraph tag